docker_services_cli/config.py

```diff
@@ -119,13 +120,33 @@
 }
 """RabbitMQ service configuration."""
 
+MINIO = {
+    "MINIO_VERSION": "MINIO_2025_LATEST",
+    # note: minio does not do semantic versioning, so we use the latest version
+    # the release at the time of writing this is RELEASE.2025-02-28T09-55-16Z
+    "DEFAULT_VERSIONS": {"MINIO_2025_LATEST": "latest"},
+    "CONTAINER_CONFIG_ENVIRONMENT_VARIABLES": {
+        "S3_ACCESS_KEY_ID": "invenio",
+        "S3_SECRET_ACCESS_KEY": "invenio8",
+    },
+    "CONTAINER_CONNECTION_ENVIRONMENT_VARIABLES": {
+        "s3": {
+            "S3_ENDPOINT_URL": "http://localhost:9000",
+            "S3_ACCESS_KEY_ID": "invenio",
+            "S3_SECRET_ACCESS_KEY": "invenio8",
```

Comment:
```suggestion
            "S3_SECRET_ACCESS_KEY": "invenio",
```

nit: unless it has to be different :)

(ditto for above)

Reply:
The password needs to be at least 8 characters long (minio constraint), that's why the '8' has been appended